src/main.ts

```diff
@@ -1,23 +1,50 @@
 import './style.css'
-import typescriptLogo from './typescript.svg'
-import { setupCounter } from './counter'
+import {Sprite} from "./view/sprite";
+import {SpaceObject} from "./model/spaceobject";
 
-document.querySelector<HTMLDivElement>('#app')!.innerHTML = `
-  <div>
-    <a href="https://vitejs.dev" target="_blank">
-      <img src="/vite.svg" class="logo" alt="Vite logo" />
-    </a>
-    <a href="https://www.typescriptlang.org/" target="_blank">
-      <img src="${typescriptLogo}" class="logo vanilla" alt="TypeScript logo" />
-    </a>
-    <h1>Vite + TypeScript</h1>
-    <div class="card">
-      <button id="counter" type="button"></button>
-    </div>
-    <p class="read-the-docs">
-      Click on the Vite and TypeScript logos to learn more
-    </p>
-  </div>
-`
+class SpaceObjectUI {
```

Comment:
Need to move this class into its own file in a new src/ui/ directory. Maybe replace src/view with that.

Reply:
Also, add a draw method

Reply:
Did it in [e06acb4](https://github.com/jhugon/orbitalGameTs/pull/5/commits/e06acb4becaffeed84e48a0098786325d3cd416a)